.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "endOfLine": "lf"
```

Comment:
Added this prettier rule to ensure consistent line endings between windows/non-windows machines. Feel free to add any others you feel strongly about